presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -77,6 +106,232 @@ protected QueryRunner createQueryRunner()
         return createIcebergQueryRunner(ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), ImmutableMap.of());
     }
 
+    @Test
+    public void testFilterPushdown()
```

Comment:
Can you please add tests for partitioned tables if it's already working?

Reply:
I am working on adding these tests as well.